index.html

```diff
@@ -2327,7 +2289,7 @@ <h4>Presentations Including Holder Claims</h4>
     },
     { <span class="comment">...</span> }
   }],
-  "proof": [{ <span class="comment">...</span> }]
```

Comment:
replace proof with ellipses, many other optional properties could be present here.

Reply:
Then we should show what a secured VP looks like.